docs/src/main/asciidoc/logging.adoc

```diff
@@ -388,8 +388,8 @@ Log4j:
 [source, xml]
 ----
 <dependency>
-    <groupId>org.jboss.logging</groupId>
-    <artifactId>jboss-logging-log4j</artifactId>
+  <groupId>org.jboss.logmanager</groupId>
+  <artifactId>log4j-jboss-logmanager</artifactId>
```

Comment:
So first let's keep the indentation consistent.

Second, I'm really not sure these are the right dependencies? They do not look consistent with the others?

Reply:
Sorry for the indentation, I made the modification on my branch but didn't push them.

These are the correct dependencies. See the first part of the guide that explain how to route log4j events to jboss-logmanager.
I assume as log4j is a log backend (the other frameworks are log facade) it is a little different and that's why it is part of jboss-logmanager and not jboss-logging.

We can wait for @dmlloyd review on this but the currently documented artifacts didn't exist.

Reply:
@gsmet this has been approve ;)
OK to merge it ?